shared_model/validators/block_validator.hpp

```diff
@@ -18,6 +18,8 @@
 #ifndef IROHA_BLOCK_VALIDATOR_HPP
 #define IROHA_BLOCK_VALIDATOR_HPP
 
+#include <boost/format.hpp>
+#include "container_validator.hpp"
```

Comment:
Please use full path as in google codestyle - `validators/container_validator.hpp`.